pkg/http/api/v1/handler_test.go

```diff
@@ -421,44 +423,56 @@ func TestRequestHandler_GetReady(t *testing.T) {
 }
 
 func TestRequestHandler_GetMetadata(t *testing.T) {
-	enqueuer := mock.NewEnqueuer()
-	store := mock.NewStore()
+	testCases := []struct {
+		name             string
+		mockedVersion    trivy.VersionInfo
+		mockedError      error
+		expectedHTTPCode int
+		expectedResp     string
+		expectedError    error
+	}{
+		{
+			name: "happy path",
+			mockedVersion: trivy.VersionInfo{
+				Trivy: "v0.5.2-17-g3c9af62",
+				VulnerabilityDB: trivy.Metadata{
+					NextUpdate: time.Unix(1584507644, 0).UTC(),
+					UpdatedAt:  time.Unix(1584517644, 0).UTC(),
+				},
+			},
+			expectedHTTPCode: http.StatusOK,
+			expectedResp: `{"scanner":{"name":"Trivy","vendor":"Aqua Security","version":"Unknown"},"capabilities":[{"consumes_mime_types":["application/vnd.oci.image.manifest.v1+json","application/vnd.docker.distribution.manifest.v2+json"],"produces_mime_types":["application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"]}],"properties":{"harbor.scanner-adapter/scanner-type":"os-package-vulnerability","harbor.scanner-adapter/vulnerability-database-next-update-at":"2020-03-18T05:00:44Z","harbor.scanner-adapter/vulnerability-database-updated-at":"2020-03-18T07:47:24Z","org.label-schema.build-date":"2019-01-03T13:40","org.label-schema.vcs":"https://github.com/aquasecurity/harbor-scanner-trivy","org.label-schema.vcs-ref":"abc","org.label-schema.version":"0.1"}}
```

Comment:
To make it easier to read, I'd format this JSON string literal and use `assert.JSONeq(t, tc.expectedResp, rr.Body.String())`

Reply:
Fixed https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/ede8e8e1b1de62f83734bc9b4e1375c1b6e62163